website/content/en/status/report-2023-07-2023-09/kde.adoc

```diff
@@ -0,0 +1,76 @@
+=== KDE on FreeBSD
+
+Links: +
+link:https://freebsd.kde.org/[KDE/FreeBSD initiative] URL: link:https://freebsd.kde.org/[] +
+link:https://community.kde.org/FreeBSD[FreeBSD -- KDE Community Wiki] URL: link:https://community.kde.org/FreeBSD[]
+
+Contact: Adriaan de Groot <kde@FreeBSD.org>
+
+The KDE on FreeBSD project packages CMake, Qt, and software from the KDE Community, for the FreeBSD ports tree.
+The software includes a full desktop environment called KDE Plasma (for both X11 and Wayland) and hundreds of applications that can be used on any FreeBSD machine.
+
+The KDE team (kde@) is part of desktop@ and x11@, building the software stack to make FreeBSD beautiful and usable as a daily-driver graphical desktop workstation.
+The notes below describe *mostly* ports for KDE, but also include items that are important for the entire desktop stack.
+
+==== Infrastructure
+
+Qt5 is now on long-term support and updates only rarely.
+There was an update to 5.15.10 in this quarter.
+Qt6 is now updated with the regular upstream releases,
+with the 6.5.2 release landing at the end of July
+and 6.5.3 following later.
+
+CMake saw no updates this quarter, so we are now lagging
+by at least one minor release. The changelog for
+the latest releases does not have much for FreeBSD,
+so there is no special reason to upgrade. 
+
+package:sysutils/polkit[] and 
+package:sysutils/consolekit[] were both updated, bringing improved
+security policy and console handling to the FreeBSD desktop.
+package:x11/sddm[] was updated to provide a better graphical
+login manager.
+
+package:multimedia/pipewire[] was updated to version 0.3.81.
+This provides multimedia supprt for desktops such as KDE and GNOME.
+
+
+==== KDE Stack
+
+KDE Gear releases happen every quarter, KDE Plasma updates once a month, and KDE Frameworks have a new release every month as well.
+These (large) updates land shortly after their upstream release and are not listed separately.
+
+* KDE Frameworks reached version 5.110. The KDE Frameworks 5 series is winding down,
+  although it will be six months or so before it enters long-term support upstream.
+* KDE Plasma Desktop was updated to version 5.27.8. Just like frameworks,
+  work on KDE Plasma 5 is winding down upstream in favor of KDE Plasma 6.
+* KDE Gear updated to 23.08.1.
+
+
+==== Related Ports
+
+The KDE ecosystem includes a wide range of ports -- most maintained by kde@,
+all building on a shared base of Qt and KDE Frameworks. The kde@ team
+updates them all as needed. This quarter, for instance, tcberner@ and arrowd@
+updated or fixed (much more than) this selection of ports:
```

Comment:
```suggestion
* KDE Frameworks reached version 5.110. The KDE Frameworks 5 series is winding down, although it will be six months or so before it enters long-term support upstream.
* KDE Plasma Desktop was updated to version 5.27.8. Just like frameworks, work on KDE Plasma 5 is winding down upstream in favor of KDE Plasma 6.
* KDE Gear updated to 23.08.1.

==== Related Ports

The KDE ecosystem includes a wide range of ports -- most maintained by kde@, all building on a shared base of Qt and KDE Frameworks.
The kde@ team updates them all as needed. This quarter, for instance, tcberner@ and arrowd@ updated or fixed (much more than) this selection of ports:
```